### ISSUE TITLE
Running pong from a remote server

### ISSUE THREAD

**Issue by miramastoras** [ramachandran-lab/pong#16]
I would like to run pong from my lab's remote HPCC cluster server, but it appears that pong is not able to create the Tornado web connection when I run it from the remote server (at least, the http://localhost:4000 address doesn't work when I open my browser, and the command stalls after telling me to open the web browser). Do you have any guidance on running pong through a remote server, or is our only option to disable the Tornado instance using --disable_server and use -v to generate files for DISTRUCT? 

Thank you very much for your help.

**Comment by abehr** [comment]
Hi Mira - Remote hosting isn't officially supported, but we are looking into adding it in a future release. Meanwhile, check out this [PR](https://github.com/ramachandran-lab/pong/pull/11), which may be of use to you.

**Comment by miramastoras** [comment]
Hi Aaron,

Thank you for your reply. I actually spoke to a colleague who pointed me to this port forwarding workaround for running on a remote server and I was able to get it to work:  

`ssh -N -L 4000:localhost:4000 <remote cluster>` 

I ran the command on my terminal, left it hanging, and then opened up a new tab and logged into the cluster and ran pong. This command setup port forwarding from my laptop’s port 4000 to the remote cluster’s port 4000.
